packages/choreography-lint/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "eslint-plugin-choreography-lint",
```

Comment:
1. It should be `@choreography-ts/eslint-plugin-choreography-lint`
2. The directory name should match the package name (`eslint-plugin-choreography-lint`)

Reply:
On second thought, I want to call this package `@choreography-ts/eslint-plugin-choreography-ts` because there can be another choreographic library in TypeScript.